### PR TITLE
Feature/ApeApi

### PR DESCRIPTION
**Pull request by TBouder**
## What it does ✨
Add a basic API for ape.tax to get the vaults for third parties.
The new router is the following : `https://ape.tax/api/vaults` and accepts `?network=[1 || 56 || 137 || 250]` as param to specify the network. Default is 1.
Only the `experimental` vaults are resolved.
Here is the schema of the resolved value. The data are cached for 10 minutes.
```json
{
	"success": true,
	"data": {
		"0": {
			"title": "Fantom's Fury",
			"logo": "👻⚡",
			"displayName": "👻⚡ Fantom's Fury",
			"src": "https://ape.tax/fantomsfury",
			"status": "active",
			"type": "experimental",
			"address": "0x36e7aF39b921235c4b01508BE38F27A535851a5c",
			"network": 250,
			"data": {
				"apiVersion": "0.3.2",
				"depositLimit": "5000000.0",
				"totalAssets": "716771.395316817876930412",
				"availableDepositLimit": "4283228.604683182123069588",
				"pricePerShare": "1.133355773675367656",
				"decimals": 18
			},
			"want": {
				"address": "0x21be370D5312f44cB42ce377BC9b8a0cEF1A4C83",
				"symbol": "WFTM",
				"cgID": "fantom"
			}
		},
		"generatedTimeMs": 1630831663952
	}
}
```

## How to test ✅
On the preview, you can try to get the `api/vaults` URI to retrieve the list of the vaults.

### CODE REVIEW

File: utils/yVault.abi.json [hunk 1]
```diff
@@ -0,0 +1 @@
...1},{"name":"debtLimit","outputs":[{"type":"uint256","name":""}],"inputs":[],"stateMutability":"view","type":"function","gas":3071},{"name":"totalDebt","outputs":[{"type":"uint256","name":""}],"inputs":[],"stateMutability":"view","type":"function","gas":3101},{"name":"lastReport","outputs":[{"type":"uint256","name":""}],"inputs":[],"stateMutability":"view","type":"function","gas":3131},{"name":"activation","outputs":[{"type":"uint256","name":""}],"inputs":[],"stateMutability":"view","type":"function","gas":3161},{"name":"rewards","outputs":[{"type":"address","name":""}],"inputs":[],"stateMutability":"view","type":"function","gas":3191},{"name":"managementFee","outputs":[{"type":"uint256","name":""}],"inputs":[],"stateMutability":"view","type":"function","gas":3221},{"name":"performanceFee","outputs":[{"type":"uint256","name":""}],"inputs":[],"stateMutability":"view","type":"function","gas":3251},{"name":"nonces","outputs":[{"type":"uint256","name":""}],"inputs":[{"type":"address","name":"arg0"}],"stateMutability":"view","type":"function","gas":3496},{"name":"DOMAIN_SEPARATOR","outputs":[{"type":"bytes32","name":""}],"inputs":[],"stateMutability":"view","type":"function","gas":3311}]
```
Comment (fameal) [inline utils/yVault.abi.json:1]: is it the same ABI for vaults 0.2.0 to 0.4.3? Just asking

**Review by fameal — APPROVED** [review]
I left I comment, but it looks good

File: utils/yVault.abi.json (hunk 1 above)
Comment (TBouder) [reply utils/yVault.abi.json:1]: We are only using a few functions from this ABI :
- `apiVersion`
- `depositLimit`
- `totalAssets`
- `availableDepositLimit`
- `pricePerShare`
- `decimals`
So it's not "so" important for this PR, but I will check and add the 0.4.3 and rename the file `yvault.043.abi.json` for the sake of clarity!

Comment (fameal) [reply utils/yVault.abi.json:1]: it's ok if you use a tiny set of functions only. No need to do it, for now 😈

**Pull request merged**